recipes/skyfield-data/LICENSE.txt

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2019 Bruno Bord, PeopleDoc Inc.
```

Comment:
Please make a PR to include this license file in the upstram sdist via the MANIFEST.in.

Reply:
I'm sorry, I've never messed with MANIFEST.in files, so I'm not sure how I would go about to do this

Reply:
No worries. Make an issue on the upstream repo and I will merge now.